src/interp/interp.cc

```diff
@@ -254,10 +254,34 @@ void Store::DeleteRoot(RootList::Index index) {
 }
 
 void Store::Collect() {
+  marks_.resize(objects_.size());
```

Comment:
Maybe add `size_t object_count = objects_.size()` which you can then use here and below on lines 269, 273 and 383.

Reply:
Good idea, done.